server/lib/accounts-manager.coffee

```diff
@@ -52,8 +52,8 @@ TryMatchAccount = (target, accounts) ->
 
         # Remove spaces (e.g. Credit Mutuel would randomly add spaces in
         # account names) and lower case.
-        oldTitle = a.title.replace(/ /g, '').toLowerCase()
-        newTitle = target.title.replace(/ /g, '').toLowerCase()
+        oldTitle = a.title.toLowerCase()
+        newTitle = target.title.toLowerCase()
```

Comment:
I had an issue with those lines while setting up my online IDE and the change got commited by error. Sorry!